server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/restore/RestoreSnapshotRequest.java

```diff
@@ -383,6 +394,26 @@ public boolean includeAliases() {
         return includeAliases;
     }
 
+    /**
+     * If set to true the logging should be at a lower level
```

Comment:
Expression is got from the similar "silent" parameter of https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/cluster/metadata/MetadataCreateIndexService.java#L416 . Not sure if it's super useful, as it does not mention what the lower level is (DEBUG or INFO). But I guessed the original description did not want to make promises about the log level, and that's why I also adopted a similar description. Feedback welcome!

Reply:
I guess if we're logging at `DEBUG` still then `silent` is too much of an absolute. Maybe we should call it `quiet` instead? And yes I agree we should specify the levels, and maybe something about what is logged too, something like this?

```suggestion
     * If {@code false}, report the completion of the restore at `INFO` level. If {@code true}, report no logs about this restore at `INFO` level.
```